src/main/resources/openapi.json

```diff
@@ -1100,6 +1100,24 @@
                 "operationId": "openapi",
                 "description": "Retrieves the OpenAPI v2 specification in JSON format."
             }
+        },
+        "/": {
+            "get": {
+                "responses": {
+                    "200": {
+                        "description": "Info about Kafka Bridge instance configuration.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    }
+                },
+                "operationId": "configuration",
+                "description": "Retrieves the Kafka Bridge instance version in JSON format."
```

Comment:
it's not just version anymore. For now, it is, but in general, it could contain more.